### ISSUE TITLE
Functional error identified in file mandels_problem.ipynb

### ISSUE THREAD

**Issue by OmarDuran** [pmgbergen/porepy#1137]
Attempts to employ `catersian` grids in the tutorial notebook produce incorrect approximations for the Mandel's problem. Possible source of the error is the combination of `mpsa`, `cartesian` grids and some kind of `BC`. The following MWE derived from the tutorial is exhibiting such functional error.
[mwe_mandels_problem.py.zip](https://github.com/pmgbergen/porepy/files/14526613/mwe_mandels_problem.py.zip)

**Comment by IvarStefansson** [comment]
Can this be closed, @OmarDuran? I believe the explanation is inherent to the discretization method, right, @keileg?

**Comment by keileg** [comment]
Yes, there's no getting around this. The only question is if we should leave an explicit warning in the tutorial not to use such a combination of methods (I'm not sure if there is such a comment now). I'm not sure I am in favor of such a change.

**Comment by IvarStefansson** [comment]
Could you please take action as you see fit and then close the issue?

**Comment by OmarDuran** [comment]
My understanding of the advantages of a multipoint finite volume method is twofold: lowest order approximations and general meshes, including cartesian meshes. @IvarStefansson  I am unable to understand how the explanation is inherent in the discretization method. Fixing this issue, however, is quite involved, and I would rather not take any action in this regard rather than leave this issue open for the record of users.

**Comment by IvarStefansson** [comment]
Sorry about the misunderstanding. My suggestion was for @keileg to add the warning if required.

**Comment by IngridKJ** [comment]
We tried reproducing. We only encountered some zero division during plotting inside the `MandelSetup`. We traced it back to division by the x component of the normal vector on the internal faces of the cells adjacent to the south boundary (obviously!). We decided to check the grid type inside the relevant plotting function, raise a warning and suppress the plot.
